website/src/content/docs/docs/howtos/Generate client libraries/02package.mdx

```diff
@@ -161,7 +166,13 @@ namespace Customization;
 ```
 
 ```typescript
-// TODO
+// model Food is updated to model Models
+const client = new PetStoreClient();
+const food:Models = {
```

Comment:
Is this expected? I don't think we have namespace concept and why this is showing as renamed ths model?